### ISSUE TITLE
how to skip CUDA libraries while installing?

### ISSUE THREAD

**Issue by srikamalteja** [pyannote/pyannote-audio#1647]
### Tested versions

- Reproducible in 3.1

### System information

macOS - 13.3 - pyannote.audio 3.1.1 M1 Pro

### Issue description

Hi Author, thanks for your work.

**This is not a bug report**

I have downloaded the model and running on CPU locally, so when I did  `pyannote.audio==3.1.1`, it installed a range of libraries as listed below. Is there a way to uninstall libraries and use particular, especially when running offline, please? 
Thanks

`aiohttp==3.9.1
aiosignal==1.3.1
alembic==1.13.1
antlr4-python3-runtime==4.9.3
asteroid-filterbanks==0.4.0
async-timeout==4.0.3
attrs==23.2.0
audioread==3.0.1
certifi==2023.11.17
cffi==1.16.0
charset-normalizer==3.3.2
click==8.1.7
colorama==0.4.6
colorlog==6.8.0
contourpy==1.2.0
cycler==0.12.1
decorator==5.1.1
docopt==0.6.2
einops==0.7.0
filelock==3.13.1
fonttools==4.47.2
frozenlist==1.4.1
fsspec==2023.12.2
greenlet==3.0.3
huggingface-hub==0.20.3
HyperPyYAML==1.2.2
idna==3.6
importlib-resources==6.1.1
Jinja2==3.1.3
joblib==1.3.2
julius==0.2.7
kiwisolver==1.4.5
lazy_loader==0.3
librosa==0.10.1
lightning==2.1.3
lightning-utilities==0.10.1
llvmlite==0.41.1
Mako==1.3.1
markdown-it-py==3.0.0
MarkupSafe==2.1.4
matplotlib==3.8.2
mdurl==0.1.2
mpmath==1.3.0
msgpack==1.0.7
multidict==6.0.4
networkx==3.2.1
numba==0.58.1
numpy==1.26.3
omegaconf==2.3.0
optuna==3.5.0
packaging==23.2
pandas==2.2.0
pillow==10.2.0
platformdirs==4.1.0
pooch==1.8.0
primePy==1.3
protobuf==4.25.2
pyannote.audio==3.1.1
pyannote.core==5.0.0
pyannote.database==5.0.1
pyannote.metrics==3.2.1
pyannote.pipeline==3.0.1
pycparser==2.21
Pygments==2.17.2
pyparsing==3.1.1
python-dateutil==2.8.2
pytorch-lightning==2.1.3
pytorch-metric-learning==2.4.1
pytz==2023.3.post1
PyYAML==6.0.1
requests==2.31.0
rich==13.7.0
ruamel.yaml==0.18.5
ruamel.yaml.clib==0.2.8
scikit-learn==1.4.0
scipy==1.12.0
semver==3.0.2
sentencepiece==0.1.99
shellingham==1.5.4
six==1.16.0
sortedcontainers==2.4.0
soundfile==0.12.1
soxr==0.3.7
speechbrain==0.5.16
SQLAlchemy==2.0.25
sympy==1.12
tabulate==0.9.0
tensorboardX==2.6.2.2
threadpoolctl==3.2.0
torch==2.1.2
torch-audiomentations==0.11.0
torch-pitch-shift==1.2.4
torchaudio==2.1.2
torchmetrics==1.3.0.post0
tqdm==4.66.1
typer==0.9.0
typing_extensions==4.9.0
tzdata==2023.4
urllib3==2.1.0
yarl==1.9.4
zipp==3.17.0
`

### Minimal reproduction example (MRE)

pyannote.audio==3.1.1

**Comment by hhd52859** [comment]
pip install "pyannote.audio==3.1.1" --no-deps
